topic-3.3/fileManagerTest.java

```diff
@@ -0,0 +1,44 @@
+package bootcamp33.topic3_3;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import bootcamp33.topic3_3.FileManager;
+
+public class fileManagerTest {
```

Comment:
[High] Test class should be named "FileManagerTest". Use uppercase for class naming.